src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -230,25 +237,20 @@ private void setLookAndFeel() {
             String systemLookFeel = UIManager.getSystemLookAndFeelClassName();
 
             if (Globals.prefs.getBoolean(JabRefPreferences.USE_DEFAULT_LOOK_AND_FEEL)) {
-                // FIXME: Problems with OpenJDK and GTK L&F
-                // See https://github.com/JabRef/jabref/issues/393, https://github.com/JabRef/jabref/issues/638
-                if (System.getProperty("java.runtime.name").contains("OpenJDK")) {
+                // FIXME: Problems with GTK L&F on Linux and Mac. Needs reevaluation for Java9
+                if (GTK_LF_CLASSNAME.equals(systemLookFeel)) {
                     // Metal L&F
-                    lookFeel = UIManager.getCrossPlatformLookAndFeelClassName();
-                    LOGGER.warn(
-                            "There seem to be problems with OpenJDK and the default GTK Look&Feel. Using Metal L&F instead. Change to another L&F with caution.");
+                    lookFeel = NIMBUS_LOOK_AND_FEEL;
+                    LOGGER.warn("There seem to be problems with OGTK Look&Feel. Using Nimbus L&F instead. Change to another L&F with caution.");
                 } else {
                     lookFeel = systemLookFeel;
                 }
             } else {
                 lookFeel = Globals.prefs.get(JabRefPreferences.WIN_LOOK_AND_FEEL);
             }
 
-            // FIXME: Open JDK problem
-            if (UIManager.getCrossPlatformLookAndFeelClassName().equals(lookFeel)
-                    && !System.getProperty("java.runtime.name").contains("OpenJDK")) {
-                // try to avoid ending up with the ugly Metal L&F
-                UIManager.setLookAndFeel("javax.swing.plaf.nimbus.NimbusLookAndFeel");
+            if (UIManager.getCrossPlatformLookAndFeelClassName().equals(lookFeel) && !GTK_LF_CLASSNAME.equals(lookFeel)) {
```

Comment:
I don't understand the second test. I thought we want to prevent the `GTK L&F` by all means....

Reply:
ok now I don't understand the complete if statement (and think we can simply delete it).

cases:
- system: gtk -> `lookFeel = Nimbus`, `if = false`
- system: win -> `lookFeel = win`, `if = true` (why do we set the l&f to nimbus?!)
- system: win, WIN_LOOK_AND_FEEL: win -> `lookFeel = win`, `if = true` (again wrong)
- system: gtk, WIN_LOOK_AND_FEEL: gtk -> `lookFeel = gtk`, `if = false` (but we actually wanted to prevent setting it to gtk?!)
- system: win, WIN_LOOK_AND_FEEL: gtk -> `lookFeel = gtk`, `if = false` (again, we don't want gtk)

If I understand you correctly, WIN_LOOK_AND_FEEL = gtk can never happen because of the migration, right? In this case we can ignore the last two cases, but even in the first three cases the behavior is odd.

Reply:
I simplified the condition. Now the cases should be more clear.